data/json/requirements/cooking_components.json

```diff
@@ -95,15 +95,12 @@
         [ "egg_bird_unfert", 1 ],
         [ "egg_chicken", 1 ],
         [ "egg_grouse", 1 ],
-        [ "egg_crow", 3 ],
-        [ "egg_raven", 3 ],
-        [ "egg_bluejay", 6 ],
-        [ "egg_cardinal", 12 ],
-        [ "egg_robin", 7 ],
-        [ "egg_pigeon", 3 ],
-        [ "egg_sparrow", 20 ],
-        [ "egg_waxwing", 20 ],
-        [ "egg_chickadee", 50 ],
```

Comment:
You're reverting my changes from my bird audit here

Reply:
I never got notified for merge conflicts and never deleted these myself?

I'll look into it. I hate git